Dockerfile

```diff
@@ -10,12 +10,24 @@ WORKDIR /usr/src
 ## Setup Home Assistant Core dependencies
 COPY requirements.txt homeassistant/
 COPY homeassistant/package_constraints.txt homeassistant/homeassistant/
+# hadolint ignore=DL4006
+RUN \ 
+   grep -v '^[#-]' < homeassistant/requirements.txt | xargs -P 6 -n 4 -- \
+   pip3 download --no-index --only-binary=:all: \
+   --find-links "${WHEELS_LINKS}" --use-deprecated=legacy-resolver \
+   --cache-dir "/tmp/pip_cache"
 RUN \
-    pip3 install --no-cache-dir --no-index --only-binary=:all: --find-links "${WHEELS_LINKS}" \
-    -r homeassistant/requirements.txt --use-deprecated=legacy-resolver
+    pip3 install --cache-dir "/tmp/pip_cache" --no-index --only-binary=:all: --find-links "${WHEELS_LINKS}" \
+    -r homeassistant/requirements.txt --use-deprecated=legacy-resolver ; exit 0
 COPY requirements_all.txt homeassistant/
+# hadolint ignore=DL4006
+RUN \ 
+   grep -v '^[#-]' < homeassistant/requirements_all.txt | xargs -P 8 -n 120 -- \
+   pip3 download --no-index --only-binary=:all: \
+   --find-links "${WHEELS_LINKS}" --use-deprecated=legacy-resolver \
+   --cache-dir "/tmp/pip_cache" ; exit 0
```

Comment:
Do we need to delete this directory after the build?

Reply:
I think thats a good idea.

0f0be7ee37297bffb52ea48ca8c35ca8ccf7cc55

Rerunning the build now to make sure it works as expected

Reply:
```
Step 12/18 : RUN rm -rf /tmp/pip_cache
 ---> Running in 447c2db84f73
```
```
 ---> 16f8f4cbe664
Successfully built 16f8f4cbe664
Successfully tagged hass/test1:latest
```
Built successfully with that change